src/main/java/org/graylog/integrations/aws/service/AWSService.java

```diff
@@ -164,34 +216,32 @@ public AvailableServiceResponse getAvailableServices() {
                                              "logs:DescribeLogGroups",
                                              "logs:PutSubscriptionFilter");
 
-        AWSPolicyStatement statement = AWSPolicyStatement.create("GraylogCloudWatchPolicy",
+        AWSPolicyStatement statement = AWSPolicyStatement.create("GraylogKinesisSetup",
                                                                  "Allow",
                                                                  actions,
                                                                  "*");
-        AWSPolicy awsPolicy = AWSPolicy.create(AWS_POLICY_VERSION, Collections.singletonList(statement));
+        return AWSPolicy.create(AWS_POLICY_VERSION, Collections.singletonList(statement));
+    }
 
-        ArrayList<AvailableService> services = new ArrayList<>();
+    /**
+     * Create the AWS Kinesis auto-setup policy.
+     */
+    private AWSPolicy buildAwsAutoSetupPolicy() {
+        List<String> actions = Arrays.asList("iam:PassRole",
+                                             "logs:CreateLogGroup",
```

Comment:
@ceruleancee Does this auto-setup policy need the `logs:CreateLogGroup`, `logs:CreateLogStream` permissions?

Reply:
good catch, no it doesnt need them